examples/dynatrace-express-apm/src/sdk-v3/sdk.js

```diff
@@ -1,46 +1,44 @@
-const path = require('path')
-const { ClientBuilder } = require('@commercetools/sdk-client-v2')
-// const { createTelemetryMiddleware } = require('@commercetools/ts-sdk-apm')
-const {
-  createTelemetryMiddleware,
-} = require('@commercetools/ts-sdk-apm/dist/commercetools-ts-sdk-apm.cjs.js')
+const { ClientBuilder } = require('@commercetools/ts-client')
+const { createTelemetryMiddleware } = require('@commercetools/ts-sdk-apm')
 const { createApiBuilderFromCtpClient } = require('@commercetools/platform-sdk')
 const fetch = require('node-fetch')
+const path = require('path')
 
 const oneagent = path.join(__dirname, '..', '..', 'oneagent-tracer.js')
+const tracer = require(oneagent)
+
 const projectKey = process.env.CTP_PROJECT_KEY
 const authMiddlewareOptions = {
-  host: 'https://auth.europe-west1.gcp.commercetools.com',
+  host: process.env.CTP_AUTH_URL,
```

Comment:
Please update the `.env.example` file to include this new variable

Reply:
you are right, I forgot to update it